grafana-plugin/src/state/rootBaseStore/index.ts

```diff
@@ -98,25 +98,28 @@ export class RootBaseStore {
   // stores
 
   async updateBasicData() {
-    const storeUpdatePromises = [
+    const updateFeatures = async () => {
+      await this.updateFeatures();
+
+      // Only fetch cloud connection status when cloud connection feature is enabled on OSS instance
+      // Note that this.hasFeature can only be called after this.updateFeatures()
+      if (this.hasFeature(AppFeature.CloudConnection)) {
+        await this.cloudStore.loadCloudConnectionStatus();
+      }
+    };
+
+    return Promise.all([
       this.teamStore.loadCurrentTeam(),
       this.grafanaTeamStore.updateItems(),
-      this.updateFeatures(),
+      updateFeatures(),
       this.userStore.updateNotificationPolicyOptions(),
       this.userStore.updateNotifyByOptions(),
       this.alertReceiveChannelStore.updateAlertReceiveChannelOptions(),
       this.alertReceiveChannelStore.updateAlertReceiveChannelOptions(),
```

Comment:
this line is duplicated

Reply:
I missed this when merging the PR and thought this is about the `updateFeatures` duplication. Anyways, here's the PR that fixes this: https://github.com/grafana/oncall/pull/1179